Sources/sentry/include/SentryApp.h

```diff
@@ -5,4 +5,28 @@
 
 #include <sentry.h>
 
-#endif
\ No newline at end of file
+#ifdef _WIN32
+#include <atomic>
```

Comment:
This module appears to be C code. Can we implement the atomic read in C?

Reply:
Perhaps but since `std::atomic` is C++, I think it needs to be C++

Reply:
In _theory_, `_Atomic` and `std::atomic` _should_ be ABI compatible, but I don't know if Microsoft conforms to the specification's suggestion.

Reply:
Ah now I understand that the runtime interface is actually specified in terms of `std::atomic`. All right!